feat(changelog): transactional-email-added-email-flags 2023-11-20 (#2421)
scaleway/docs-content: changelog/november2023/2023-11-20-transactional-email-added-email-flags.mdx

---
title: Email Flags
status: added
author:

>     fullname: 'Join the #transactional-email on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #transactional-email channel on Slack.'
```

>     url: 'https://scaleway-community.slack.com/archives/C0358JE0XEK'

Reviewer: ```suggestion
    url: 'https://slack.scaleway.com'
```

date: 2023-11-20
category: managed-services
product: transactional-email
---

> Take control of delivery errors with email flags. Whether it's a Hard bounce, Soft bounce, Mailbox full, or other issues, these flags provide immediate insights into the reasons behind delivery failures.

Reviewer: ```suggestion
Take control of delivery errors with email flags. Whether it's a "hard bounce", "soft bounce", "mailbox full", or other issues, these flags provide immediate insights into the reasons behind delivery failures.
```

> You can filter on these flags via the API and the Activity Console tab.

Reviewer: ```suggestion
You can filter on these flags via the API and the from the "Activity" tab in the Scaleway console.
```

> List here https://www.scaleway.com/en/docs/managed-services/transactional-email/concepts/#flags

Reviewer: ```suggestion
For more inofrmation, refer to the [Transactional Email flags concept page](https://www.scaleway.com/en/docs/managed-services/transactional-email/concepts/#flags).
```
